_posts/2015-09-18-epages-you-2015.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Annual company event: ePagees rock the YOU"
+date: "2015-09-18 07:21:00"
+icon: users
+categories: events
+authors: ["Birgit"]
+---
+
+Creative team presentations, team building activities, workshops and lot's of fun: last weeks' activities and corridor talks at ePages all revolved around the ePages annual company event: the YOU.
+The YOU is a formal, annual company event which all ePages employees are required to attend.
+This sounds really tough.
+But it is not.
+It is incredibly fun, although it is hard work, too and involves long hours.
+
+Main goal of the event:
+
+* get to know each other better
+* unite colleagues from all company locations
+* exchange ideas among colleagues, departments and countries.
+
+# Event task force
+
+Already a long time before the actual date of the YOU, an event task force started their activities to prepare the ground for two awesome days.
+To make it a big surprise for everyone, the agenda was kept a secret for a long time.
+But soon the time had come to position some teasers and involve the employees.
```

Comment:
However, leading up to the YOU we were given teasers for the event which kept us all excited.